android/modules/ui/src/java/ti/modules/titanium/ui/WindowProxy.java

```diff
@@ -594,4 +595,11 @@ public String getApiName()
 	{
 		return "Ti.UI.Window";
 	}
+
+	private void logDeprecatedPropertiesWarning() {
```

Comment:
the "please use" i believe wont work as the properties exist, however are not being use in the method call and so  would error so for now i will just print a warning about the depreciation of the animation properties